wandb/cli/cli.py

```diff
@@ -1185,9 +1264,20 @@ def launch_agent(
             "You must specify a project name or set WANDB_PROJECT environment variable."
         )
 
-    wandb.termlog("Starting launch agent ✨")
+    if sweep_id is not None:
+        # TODO: This is not the final resting place for this (do not worry)
+        # the launch scheduler will become a type of Job (Launch Jobs API)
+        # until then this felt like the least worst way to start launch schedulers.
+        wandb.termlog("Starting a Launch Scheduler 🚀")
+        from wandb.sdk.launch.sweeps import load_scheduler
 
-    wandb_launch.create_and_run_agent(api, agent_config)
+        _scheduler = load_scheduler("sweep")(
+            api, entity=entity, project=project, queue=queues[0], sweep_id=sweep_id
+        )
+        _scheduler.start()
```

Comment:
if this is being started in a second process, will the launch agent have a way to kill it on KeyboardInterupt?